charts/kong/UPGRADE.md

```diff
@@ -57,6 +58,19 @@ text ending with `field is immutable`. This is typically due to a bug with the
 If you encounter this error, deleting any existing `init-migrations` jobs will
 clear it.
 
+## 2.1.0
+
+### Migration off Bintray
```

Comment:
This happened in 2.0.0, didn't it?

I think that adding the remark about this change to 2.1.0 (while migration of bintray happened in the previous minor release) may be a source of confusion.

Reply:
Partially. We didn't get everything in the last release, and the 2.1 defaults are the first set that has the new repos for all images.